src/site/stages/build/index.js

```diff
@@ -175,7 +221,17 @@ function defaultBuild(BUILD_OPTIONS) {
 
   smith.use(downloadDrupalAssets(BUILD_OPTIONS), 'Download Drupal assets');
 
-  configureAssets(smith, BUILD_OPTIONS);
+  if (BUILD_OPTIONS['asset-source'] !== assetSources.LOCAL) {
```

Comment:
Does this need to be added to `src/site/stages/build/options.js`?

Reply:
It's [already there](https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/site/stages/build/options.js#L31).